cpp/ql/src/Best Practices/Unused Entities/UnusedStaticVariables.qhelp

```diff
@@ -11,7 +11,7 @@ caused by an unhandled case.</p>
 
 </overview>
 <recommendation>
-<p>Check that the unused static variable does not indicate a defect, for example, an unhandled case. If the static variable is genuinuely not needed, 
+<p>Check that the unused static variable does not indicate a defect, for example, an unhandled case. If the static variable is genuinely not needed, 
```

Comment:
I'm assuming this (and similar items) is (are) user facing.

Reply:
Yes, `.qhelp` files are used to generate HTML, for example you can see this mistake here: https://codeql.github.com/codeql-query-help/cpp/cpp-unused-static-variable/